src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClientX/Handlers/Connection/ConnectionHandlerOrchestrator.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.Data.SqlClientX.Handlers.Connection
+{
+    internal static class ConnectionHandlerOrchestrator
```

Comment:
Renamed to Connection Handler orchestrator, and made static. I am looking forward to SqlConnector to own the ConnHandlerContext and pass on to Orchestrator to process connection request.